python/cuml/dask/ensemble/randomforestregressor.py

```diff
@@ -129,6 +126,7 @@ def __init__(
         max_leaves=-1,
         n_streams=4,
         accuracy_metric="mse",
+        dtype=None,
```

Comment:
I should have looked through the whole files a bit more closely, rather than jjust reviewing the changes. I notice both the classifier and regressor code still does the following in the constructor. Can we consolidate these into the base class's constructor?

```
        unsupported_sklearn_params = {
            "criterion": criterion,
            "min_samples_leaf": min_samples_leaf,
            "min_weight_fraction_leaf": min_weight_fraction_leaf,
            "max_leaf_nodes": max_leaf_nodes,
            "min_impurity_decrease": min_impurity_decrease,
            "min_impurity_split": min_impurity_split,
            "oob_score": oob_score,
            "n_jobs": n_jobs,
            "random_state": random_state,
            "warm_start": warm_start,
            "class_weight": class_weight,
        }

        for key, vals in unsupported_sklearn_params.items():
            if vals is not None:
                raise TypeError(
                    "The Scikit-learn variable",
                    key,
                    " is not supported in cuML,"
                    " please read the cuML documentation for"
                    " more information",
                )

        self.n_estimators = n_estimators
        self.n_estimators_per_worker = list()
        self.num_classes = 2

        self.client = default_client() if client is None else client
        if workers is None:
            workers = self.client.has_what().keys()  # Default to all workers
        self.workers = workers
        self._create_the_model(
            model_func=RandomForestClassifier._func_build_rf,
            max_depth=max_depth,
            n_streams=n_streams,
            max_features=max_features,
            n_bins=n_bins,
            split_algo=split_algo,
            split_criterion=split_criterion,
            min_rows_per_node=min_rows_per_node,
            bootstrap=bootstrap,
            bootstrap_features=bootstrap_features,
            type_model=type_model,
            verbose=verbose,
            rows_sample=rows_sample,
            max_leaves=max_leaves,
            quantile_per_tree=quantile_per_tree,
            dtype=dtype)
```